BUILDING.md

```diff
@@ -1,44 +1,44 @@
 # Building JanusGraph client for Python
 
-- Building Library:
+  - Building Library:
 
-A set of scripts have been created to make the job of building the library easier for user, but the scripts
-expects few pre requisites to be installed before hand. 
+  A set of scripts have been created to make the job of building the library easier for user, but the scripts
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-content-indent] Don’t use mixed indentation for children, add 2 spaces](https://app.codacy.com/manual/JanusGraph/janusgraph-python/pullRequest?prid=4792911)